shared/editor/nodes/Image.tsx

```diff
@@ -159,6 +159,31 @@ export default class Image extends SimpleImage {
     };
   }
 
+  get plugins() {
+    return [
+      new Plugin({
+        props: {
+          handleKeyDown: (view, event) => {
+            // prevent prosemirror's default spacebar behavior
+            // & zoom in if the selected node is image
+            if (event.key === " ") {
+              const { state } = view;
+              const { selection } = state;
+              if (selection instanceof NodeSelection) {
+                const { node } = selection;
+                if (node.type.name === "image") {
+                  return true;
```

Comment:
@tommoor how should i use existing zoom functionality which is already there.
Tried to copy handleSelect's code which executes on click, but didn't work.

Reply:
This might be a case where simulating a click on the element is the best way to go.